extension.neon

```diff
@@ -3,19 +3,15 @@
 # These can be reused by third party packages by including 'vendor/composer/pcre/extension.neon'
 # in your phpstan config
 
-conditionalTags:
-    Composer\Pcre\PHPStan\PregMatchParameterOutTypeExtension:
-        phpstan.staticMethodParameterOutTypeExtension: %featureToggles.narrowPregMatches%
-    Composer\Pcre\PHPStan\PregMatchTypeSpecifyingExtension:
-        phpstan.typeSpecifier.staticMethodTypeSpecifyingExtension: %featureToggles.narrowPregMatches%
-    Composer\Pcre\PHPStan\UnsafeStrictGroupsCallRule:
-        phpstan.rules.rule: %featureToggles.narrowPregMatches%
```

Comment:
btw: I think this conditional tag didn't work as expected because we always the rule at line 21.
to make this conditional-tag work, we should have registered the rule as a service without a tag (and not below the `rules:` section)